### DISCUSSION TITLE
nomacs.org website offline for a few weeks

### DISCUSSION THREAD

**Discussion by CSA-IT-Team** [nomacs/nomacs#989] [General]
Hi Folks, not sure if it's OK to post here - nomacs.org was functional a few weeks back but has now been offline for a while. Is it coming back?

**Comment by novomesk** [comment]
Most probably it is not coming back. It is a website owned by previous nomacs developers and the community doesn't have access to that infrastructure. Even if it comes back, it cannot be updated.

**Comment by CSA-IT-Team** [comment]
Thanks for your response and help for this :)
